src/Integration.UnitTests/WPF/IsValidOrganizationKeyConverterTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Globalization;
+using System.Windows;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Integration.Connection.UI;
+using SonarLint.VisualStudio.Integration.WPF;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests.WPF
+{
+    [TestClass]
+    public class IsValidOrganizationKeyConverterTests
+    {
+        [TestMethod]
+        public void Convert_NonBoolTargetType_ThrowsArgumentException()
```

Comment:
Unit tests for the new class that takes a string and returns whether it is a valid organization key.